README.md

```diff
@@ -44,7 +46,17 @@ The application will be available in your browser at `http://localhost:3000`.
 
 Because the application makes HTTP using Javascript, CORS needs to be implemented on your ArchivesSpace instance. See [as-cors](https://github.com/RockefellerArchiveCenter/as-cors) for an example of how to do this with an ArchivesSpace plugin.
 
-## Installation
+## Deploying in production
+
+In order to deploy in production, `dm_log` expects the following environment variables to be set (the values associated with each of these may change):
+
+```
+MYSQL_ROOT_PASSWORD=example
+SECRET_KEY_BASE=Wa4Kdu6hMt3tYKm4jb9p4vZUuc7jBVFw
```

Comment:
Might be worth adding a note as to why this is not needed when using `docker-compose.yml`?